users/user-manager.js

```diff
@@ -3,26 +3,26 @@ var userModels = require('./user-models.js');
 var Student = userModels.Student;
 var Admin = userModels.Admin;
 
-var createUser = (json) => {
-  if (json.role == 'Student') {
+var createUser = (reqData, createCallback) => {
+  if (reqData.role == 'Student') {
     // Create new student.
     var student = new Student({
       approvalStatus: false,
-      email: json.email,
-      firstName: json.firstName,
-      lastName: json.lastName,
+      email: reqData.email,
+      firstName: reqData.firstName,
+      lastName: reqData.lastName,
       points: 0,
```

Comment:
Points is now an object of key value pairs where keys are the categories and values are the points for that category. However, I believe we can safely assume that all users start out with 0 points in each category. Therefore instead of needing that in the `reqData`, initialize it yourself like you did `approvalStatus`